### DISCUSSION TITLE
Welcome to ros-apartment Discussions!

### DISCUSSION THREAD

**Discussion by mnovelo** [rails-on-services/apartment#294] [Announcements]
## 👋 Welcome to the Apartment Community!

We’re excited to open Discussions as a place to connect with developers and contributors who use `ros-apartment` to bring multitenancy to their Rails applications. Here, you can:

* **Ask questions** about implementing or customizing multitenancy in your Rails app.
* **Share ideas** for new features or improvements to make `ros-apartment` even more versatile.
* **Engage with other community members** by sharing tips, best practices, and experiences.
* **Welcome and support others** as we work together to build a thriving community.

Whether you’re a seasoned contributor, a first-time user, or just exploring `ros-apartment` for your project, we’re glad you’re here! To get started, introduce yourself below and let us know how you’re using `ros-apartment` in your Rails applications.

**Comment by richasinha-creator** [comment]
I have been using ros-apartment since past 3 years, we are on an older version. We are planning to move to the latest version of rails but we do not have the compatibility with the latest version of ros-apartment(3.1.2). when can we expect a newer version supporting Rails 8.1.x

**Reply by mnovelo** [reply]
I'm hoping by EOY! The new version will be a complete rewrite. You can review that progress at https://github.com/rails-on-services/apartment/pull/327
